### DISCUSSION TITLE
最小化浮窗能增加一个不置顶的选项吗

### DISCUSSION THREAD

**Discussion by happytbs** [Class-Widgets/Class-Widgets#299] [Feature]
### 检查清单

- [X] 我已更新到最新版 ![最新的版本号](https://img.shields.io/github/v/release/RinLit-233-shiroko/Class-Widgets?include_releases&style=flat-square&label=) ，并看过[最新提交](https://github.com/RinLit-233-shiroko/Class-Widgets/commits/main/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/RinLit-233-shiroko/Class-Widgets/issues?q=label%3ABug) 中检索，确认这一 Bug 未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 **软件本来** 的开发目标，而非 **添加与课表及教学辅助无关的内容** 。
- [X] 我已知晓由于开发者时间精力有限，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

全屏窗口的情况下悬浮窗（置顶状态下）仍然会遮挡住全屏显示的内容（如果能贴边隐藏也可以）

### 想要实现或优化的功能

最小化的浮窗能不置顶并且被其他窗口所覆盖或者贴边隐藏悬浮窗

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

### 不要乱下载别人发的文件，特别是名字是fix.zip的文件。您是否遇见过这样的文件？

否

**Comment by RinLit-233-shiroko** [comment]
![image](https://github.com/user-attachments/assets/4fa2cf2e-5efb-4bb6-8570-1d827a8a23fe)
可以贴边隐藏的哦，请在“设置”->“高级选项”中选择您希望的隐藏方式。

**Comment by happytbs** [comment]
> ![image](https://private-user-images.githubusercontent.com/177950371/380746261-4fa2cf2e-5efb-4bb6-8570-1d827a8a23fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMjQ1NDQsIm5iZiI6MTczMDEyNDI0NCwicGF0aCI6Ii8xNzc5NTAzNzEvMzgwNzQ2MjYxLTRmYTJjZjJlLTVlZmItNGJiNi04NTcwLTFkODI3YThhMjNmZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOFQxNDA0MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NzI5YjgzZTY5MjUxMzQ1NjRjNzVlYmU0ZDFhZmU1NDBjZjkwMmVhOTgzOGMwNDUwYjEyNGY3MzcyZWZhNjY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.8NxTACQi41xUac_m1uLsiOJuwdq4_Pp95Yrb05kPxTg) 可以贴边隐藏的哦，请在“设置”->“高级选项”中选择您希望的隐藏方式。

额，我的意思是选择最小化的悬浮窗之后的悬浮窗能够自动贴边隐藏（主要是还是感觉挡着屏幕……）

**Comment by RinLit-233-shiroko** [comment]
az这样啊，好吧
